run/archiveloop

```diff
@@ -274,8 +274,24 @@ function archive_teslacam_clips () {
 
   fix_errors_in_cam_file
 
+  if [ -f /root/bin/tesla_api.py ]
+  then
+    while :
+    do
+      sleep 120
```

Comment:
After turning the car off, will it stay awake long enough to get past this initial sleep, or should it call tesla_api first, end *then* sleep?

Reply:
I don't think it would have any effect, since by the time wifi is connected, you're still likely driving the car. It will take a lot longer than two minutes before it goes to sleep - likely around 15 minutes.